GRANTS.yaml

```
@@ -67,6 +67,17 @@ CINECA-Project:
     url: https://www.cineca-project.eu
 
 
+dekcd:
+    name: de.kcd
+    github: https://github.com/deKCD
+    joined: ?
```

This is used to indicate when they were added to the GTN, so can just set it to this month:

```suggestion
    joined: 2025-04
```